packages/terra-card/src/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin terra-consumer-theme-card() {
+  --terra-card-box-shadow: 0 1px 1px 0 #1c1f21;
+  --terra-card-border: '';
+}
+@mixin terra-consumer-theme-margin() {
+  --terra-card-margin: 10px;
```

Comment:
i don't think we want a margin on our cards..... This should be controlled by whatever lays them out